perf.groovy

```diff
@@ -341,15 +408,18 @@ def static getOSGroup(def os) {
                 }
             }
             def osGroup = getOSGroup(os)
-            def architecture = 'x64'
-            def configuration = 'Release'
             def runType = isPR ? 'private' : 'rolling'
             def benchViewName = isPR ? 'coreclr private \$BenchviewCommitName' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
 
             steps {
                 shell("bash ./tests/scripts/perf-prep.sh --throughput")
```

Comment:
Thinking it over--I think I'd like to avoid changing this for now, and keep it as currently written in this PR. As long as we don't actually execute any of the corefx binaries built here, where they get built shouldn't matter to perf. I'd be inclined to make the change if it was to be kept long-term, but I'd rather defer to @adiaaida's planned fix instead.